app/furniture/journal/room.rb

```diff
@@ -0,0 +1,7 @@
+class Journal::Room < Room
+  has_many :journal_entries, inverse_of: :room, class_name: 'Journal::Entry'
```

Comment:
Is the only purposes of this new Room class to have the `:journal_entries` association?

I don't love it. It seems confusing to me, overloaded with our Room-Room, and causing us to have bits of code like `room.becomes(Room)...`.

If the main purpose of this is to load the `entries` in `Journal` (without adding a `has_many` to our proper Room), I'd prefer we removed this class and instead implemented `entries` with an ActiveRecord lookup, like `Entry.where(room_id: room).recent`.

Reply:
Ya, this gets wiped in the fancier version of the PR.